ddtrace/context.py

```diff
@@ -20,27 +18,33 @@
 log = get_logger(__name__)
 
 
-@attr.s(eq=False, slots=True)
```

Comment:
should we add in a `__str__` and a `__repr__` ?

Reply:
Good catch, yeah we should add these back in